tests/stub/routing/test_routing_v5x0.py

```diff
@@ -2796,7 +2796,7 @@ def test_should_drop_connections_failing_liveness_check(self):
     @driver_feature(types.Feature.TMP_DRIVER_MAX_CONNECTION_POOL_SIZE,
                     types.Feature.API_CONNECTION_ACQUISITION_TIMEOUT)
     def test_should_enforce_pool_size_per_cluster_member(self):
-        acq_timeout_ms = 100
+        acq_timeout_ms = 2000
```

Comment:
Have you played around with the value? The bigger it is, the longer the test takes. So while we want it to be big enough to not be flaky, we also want it to be as small as possible.